package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-team-calendar",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
This isn't published to NPM so it shouldn't be needed

Reply:
Okay, so there is nothing in the build pipeline that needs a new version? If you can confirm, I'll remove.

Reply:
@nschonni any response?

Reply:
I think he was saying this number is only used by npm. Extension version would be specified somewhere else.  The current version of  Team Calendar is/was completely different than 0.0.2.

Reply:
@pabrams so what is needed for this PR exactly?